project.py

```diff
@@ -81,6 +92,34 @@ def find_board(self, image):
 
         return our_cnt
 
+    def seed_filter(self, image, seed):
+        """
+        My ghetto green filter that only checks if we really hit board.
+        So it checks if the green value of seed pixel is the highest.
+        """
+        rgb = image[self.seed[1], self.seed[0]]
+        if rgb[1] > rgb[0] and rgb[1] > rgb[2]:
+            return True
+        return False
+
+    def get_seed(self, image):
+        """
+        Tries to find good seed for flood fill algorithm by randomly picking
+        a point somewhere in the middle of a image and then checking if its
+        value is correct.
+        """
+        if self.seed is not None and self.seed_filter(image, self.seed):
+            return self.seed
+
+        h, w = image.shape[:2]
+        self.seed = (randint(w/4, (w/4)*3), randint(h/4, (h/4)*3))
+        # three retries to find proper seed
+        for i in xrange(1, 3):
```

Comment:
@Foxiy what is the point in doing this three times if nor the `self.seed`, neither the `image` changes?